### PR TITLE
[Packaging] Make OSX wheel build verbose

### PR DESCRIPTION
**Pull request by kszucs**
[build-239-wheel-osx-cp36m](https://travis-ci.org/kszucs/crossbow/builds/405580361)

### CODE REVIEW

File: python/setup.py [hunk 1]
```diff
@@ -242,7 +242,7 @@ def _run_cmake(self):
             else:
                 build_tool_args.append('--')
                 if os.environ.get('PYARROW_BUILD_VERBOSE', '0') == '1':
-                    build_tool_args.append('VERBOSE=1')
+                    cmake_options.append('-DCMAKE_VERBOSE_MAKEFILE=ON')
```
Comment (kszucs) [inline python/setup.py]: @wesm to support both `make` and `ninja`

**Comment by wesm** [comment]
Awesome thanks

**Comment by kszucs** [comment]
BTW I'm located in central europe, not NY :)

**Review by wesm — APPROVED** [review]
+1

**Pull request merged**